File: iOS ObjectiveC/1. Introduccion programación Objective C/Ejemplos/EjemploInterfaz/EjemploInterfaz/ViewController.m
//
//  ViewController.m
//  EjemploInterfaz
//
//  Created by Julio Laptop on 11/19/16.
//  Copyright © 2016 Julio Galeano. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)clickAction:(id)sender {
    NSString *a = [sender tag]==0?@"a":@"b";
    NSLog(@"%@",a);
}

@end

File: Swift/AudioEjemplo/AudioEjemplo/ViewController.swift
//
//  ViewController.swift
//  AudioEjemplo
//
//  Created by Julio Cesar Galeano on 10/10/19.
//  Copyright © 2019 Julio Cesar Galeano. All rights reserved.
//

import UIKit
import AVFoundation

class ViewController: UIViewController {

    var audioPlayer: AVAudioPlayer?
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }

    @IBAction func playClick(_ sender: Any) {
        do {
            if let fileURL = Bundle.main.path(forResource: "Cancion", ofType: "mp3") {
                audioPlayer = try AVAudioPlayer(contentsOf: URL(fileURLWithPath: fileURL))
                audioPlayer?.play()
            } else {
                print("No file with specified name exists")
            }
        } catch let error {
            print("Can't play the audio file failed with an error \(error.localizedDescription)")
        }
    }
    
    @IBAction func pauseClick(_ sender: Any) {
        audioPlayer?.pause()
        audioPlayer?.stop()
    }
}

